net/http/src/THttpLongPollEngine.cxx

```diff
@@ -79,37 +154,40 @@ void THttpLongPollEngine::SendCharStar(const char *buf)
 /// function called in the user code before processing correspondent websocket data
 /// returns kTRUE when user should ignore such http request - it is for internal use
 
-Bool_t THttpLongPollEngine::PreviewData(THttpCallArg &arg)
+Bool_t THttpLongPollEngine::PreviewData(std::shared_ptr<THttpCallArg> &arg)
 {
-   if (!strstr(arg.GetQuery(), "&dummy")) {
+   if (!strstr(arg->GetQuery(), "&dummy")) {
       // this is normal request, deliver and process it as any other
       // put dummy content, it can be overwritten in the future
-      arg.SetContentType("text/plain");
-      arg.SetContent(gLongPollNope);
+      arg->SetTextContent(std::string(gLongPollNope));
       return kFALSE;
    }
 
-   if (&arg == fPoll) {
-      Error("PreviewData", "NEVER SHOULD HAPPEN");
-      gSystem->Exit(12);
-   }
+   if (arg == fPoll)
+      Fatal("PreviewData", "Submit same THttpCallArg object once again");
```

Comment:
`R__FATAL_HERE` - and possibly below, I'll stop commenting on this :-)

Reply:
No - it is code for ROOT6 also